### PR TITLE
Add tcld user commands

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

Adds `tcld user *` commands.
like:
delete, get, invite, list, resend-invite, set-accout-role, set-namespace-permissions,

https://github.com/temporalio/tcld/pull/162

- EDU-591

### CODE REVIEW

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The `tcld user set-account-role` command sets an account role for the specified user in Temporal Cloud.
+You must set either `--user-email` or `--user-id`.
+
+Alias: `ri`
+
+The following modifiers control the behavior of the command.
+
+### `--user-email`
+
+Specify the user's email to set an account role to.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-account-role.md:19]:
```suggestion
Specify the email address of the user to assign an account-level Role to.
```

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+**Example**
+
+```command
+tcld user set-account-role --user-email <test@example.com> --account-role Developer
+```
+
+### `--user-id`
+
+Specify the user's identifier to set an account role to.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-account-role.md:31]:
```suggestion
Specify the user identifier of the user to assign an account-level Role to.
```

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+The request identifier to use for the asynchronous operation.
+
+If not set, the server assigns an identifier.
+
+Alias: `-r`
+
+### `--resource-version`
+
+Specify a resource version (ETag) to update from. If not specified, the latest version is used.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-account-role.md:51]:
```suggestion
Specify a resource version (ETag) to update from.
If not specified, the latest version is used.
```

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Alias: `-v`
+
+### `--account-role`
+
+_Required modifier_
+
+Specify the account role to set on the user.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-account-role.md:59]:
```suggestion
Specify the account-level Role to assign to the user.
```

File: docs-src/cloud/tcld/user/resend-invite.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+id: resend-invite
+title: tcld user resend-invite
+sidebar_label: resend-invite
+description: How to resend invitations to users in Temporal Cloud using tcld.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/resend-invite.md:5]:
```suggestion
description: How to resend an invitation to a user in Temporal Cloud using tcld.
```

File: docs-src/cloud/tcld/user/set-namespace-permissions.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+id: set-namespace-permissions
+title: tcld user set-namespace-permissions
+sidebar_label: set-namespace-permissions
+description: How to set a Namespace permission for a user in Temporal Cloud using tcld.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-namespace-permissions.md:5]:
```suggestion
description: How to set Namespace-level permissions for a user in Temporal Cloud using tcld.
```

File: docs-src/cloud/tcld/user/set-namespace-permissions.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+tags:
+  - tcld
+---
+
+The `tcld user set-namespace-permissions` command sets Namespace permissions for a specified user in Temporal Cloud.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-namespace-permissions.md:10]:
```suggestion
The `tcld user set-namespace-permissions` command sets [Namespace-level permissions](/cloud/#namespace-level-permissions) for a specified user in Temporal Cloud.
```

File: docs-src/cloud/tcld/user/set-namespace-permissions.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+You must set either `--user-email` or `--user-id`.
+
+Alias: `snp`
+
+The following modifiers control the behavior of the command.
+
+### `--user-email`
+
+Specify the user's email to set Namespace permissions to.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-namespace-permissions.md:19]:
```suggestion
Specify the email address of the user to assign Namespace-level permissions to.
```

File: docs-src/cloud/tcld/user/set-namespace-permissions.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+**Example**
+
+```command
+tcld user set-namespace-permissions --user-email <test@example.com>
+```
+
+### `--user-id`
+
+Specify the user's identifier to set Namespace permissions to.
```
Comment (djmagee) [inline docs-src/cloud/tcld/user/set-namespace-permissions.md:29]:
```suggestion
Specify the user identifier of the user to assign Namespace-level permissions to.
```

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+### `--request-id`
+
+The request identifier to use for the asynchronous operation.
+
+If not set, the server assigns an identifier.
+
+Alias: `-r`
+
+### `--resource-version`
```
Comment (tminusplus) [inline docs-src/cloud/tcld/user/set-account-role.md:49]: `--resource-version` is also supported in `set-namespace-permissions` and `delete`.

File: docs-src/cloud/tcld/user/set-account-role.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+Specify a resource version (ETag) to update from.
+If not specified, the latest version is used.
+
+Alias: `-v`
+
+### `--account-role`
+
+_Required modifier_
```
Comment (tminusplus) [inline docs-src/cloud/tcld/user/set-account-role.md:58]: I wonder should the required args go to the top of the documentation?

**Review by tminusplus — APPROVED** [review]
🚀 thank you for writing this! Looks good, left a few small comments otherwise LGTM.

**Review by tminusplus — APPROVED** [review]
:shipit:

**Review by djmagee — APPROVED** [review]

**Pull request merged**